src/PlayerBoard.js

```diff
@@ -0,0 +1,49 @@
+import { formatGridAttributes } from './formatGridAttributes';
+
+export function PlayerBoard({ ships, opponentShoots, opponentHits }) {
```

Comment:
You can extract components, or reuse the ones extracted from the `<OpponentBoard>`.